documentation/source/development/getting-started.rst

```diff
@@ -0,0 +1,67 @@
+Getting started
+===============
+
+Required Software
```

Comment:
Would move this section above optional software.

Reply:
It already is? Optional software comes later.

Reply:
I meant directly before it.

Reply:
Oh yes now I see it. Right.

Reply:
Done.